src/components/timecard/start-finish-time-input/StartFinishTimeInput.jsx

```diff
@@ -91,11 +95,18 @@ const StartFinishTimeInput = ({
               errors={errors}
               defaultValue={finishTimeValue}
               register={register}
-              autoComplete="off"
-              type="text"
+              getFormValues={getFormValues}
+              timeEntry={timeEntry}
+              timeEntriesIndex={timeEntriesIndex}
             />
           </div>
         </div>
+        <p
+          id="end-next-day"
+          className="govuk-hint govuk-grid-column-one-third govuk-!-padding-top-8 govuk-!-padding-right-0 govuk-!-padding-left-0"
+        >
+          {timeEntry.finishNextDay ? 'Finishes next day' : ''}
```

Comment:
Although the ticket description didn't mention any specific text, the mockup suggests the following text:

> Shift ending on next day

Perhaps worth a change..

Reply:
See ticket 1724 - https://collaboration.homeoffice.gov.uk/jira/browse/EAHW-1724
I spoke with Olivia and confirmed her comment on that ticket to be correct.

Reply:
Ah yes, didn't see that comment.